src/EnergyPlus/WaterUse.cc

```diff
@@ -992,7 +992,7 @@ namespace WaterUse {
             this->setupMyOutputVars = false;
         }
 
-        if (allocated(DataPlant::PlantLoop) && !this->StandAlone) {
+        if (this->plantScanFlag && allocated(DataPlant::PlantLoop) && !this->StandAlone) {
```

Comment:
I verified this loop was getting hit often by adding a print statement and running an annual simulation.  The print output came out zillions of times (approximately).  I simply added this flag so that this call will only be made once per instance of a water use connection.